src/components/cylc/Header.vue

```diff
@@ -47,26 +47,70 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
           rules: [(val) => Boolean(val) || 'Required'],
         }
       }">
+        <!-- Owner combobox -->
         <v-combobox
           class="w-100"
           id="cylc-owner-combobox"
           :disabled="store.state.user.user.mode !== 'multi user'"
           label="server owner"
           :default="owner"
-          :items="Array.from(owners)"
+          :items="owners"
           v-model="owner"
-          @keyup.enter="owners.add(owner)"
-        />
+          @keyup.enter="addOwner(owner)"
+        >
+          <template v-slot:selection="data">
+            <!-- HTML that describe how select should render selected items -->
+            {{ data.item.title }}
+          </template>
+          <template v-slot:item="{ item, props }">
+            <!-- HTML that describe how select should render items when the select is open -->
+            <div class="d-flex">
+              <v-list-item
+                class="flex-grow-1"
+                :title="item.title"
+                @click="props.onClick"
+                >
+              </v-list-item>
+              <button v-if="item.title != ownerOnLoad" @click="removeOwner(item.title)" class="mx-2">
+                <v-icon class="my-auto" color="pink-accent-4">
+                  {{ mdiCloseCircle }}
+                </v-icon>
+              </button>
+            </div>
```

Comment:
Can simplify this. Here I'm using the `append` slot of the `v-list-item` so Vuetify takes care of the positioning.
```suggestion
            <v-list-item
              :title="item.title"
              v-bind="props"
            >
              <template v-slot:append v-if="item.title !== ownerOnLoad">
                <v-icon
                  @click.stop="removeOwner(item.title)"
                  :icon="mdiClose"
                />
              </template>
            </v-list-item>
```